src/components/App/App.jsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import Lyric from '../Lyrics/Lyric';
```

Comment:
Might be easier to keep track of your component/pathing names if you used more distinctive labels.